packages/manager/modules/dedicated-universe-components/src/ovh-angular-http/ovh-angular-http.module.js

```diff
@@ -1,8 +1,12 @@
 import set from 'lodash/set';
 
-angular.module('App').config((OvhHttpProvider, constants) => {
+const moduleName = 'ducOvhAngularHttp';
```

Comment:
i'm not sure it should be a module, it's just some configuration of OvhHttp
is it reused in a lot of app/modules ?

Reply:
yes its used throughout dedicated and also some components in this module

Reply:
OvhHttp is already configured in core : https://github.com/ovh/manager/blob/master/packages/manager/modules/core/src/index.js#L231

Reply:
The first line is commented in manager-core. Looks like its not used. I can safely use the manager-core version.

// OvhHttpProvider.rootPath = constants.swsProxyPath;